domain/credential/types.go

```diff
@@ -3,79 +3,6 @@
 
 package credential
 
-import (
-	"fmt"
-
-	"github.com/juju/errors"
-	"github.com/juju/names/v5"
-)
-
-// ID represents the id of a cloud credential.
-type ID struct {
```

Comment:
Core can import domain, so we need to lift this out of domain.

Reply:
THANK YOU!!!